drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -472,6 +473,21 @@
   font-size: 0.875em;
 }
 
+.xblock--drag-and-drop .btn-link {
+    padding: 1px;
+    font-size: 0.875em;
+}
+
+.xblock--drag-and-drop .btn-default.btn-link:hover,
```

Comment:
@cahrens `unbotton` class sets border to none. I'm not 100% sure, but it seems like `unbotton` does not do anything now, but at some moment it was used to "unapply" edX button styling from buttons.

Before we start discussing it I'd suggest we don't hijack this PR for going over the styling and removing dead/useless rules. If that's required we can have a separate "CSS cleanup" task/story.